zvariant/src/str.rs

```diff
@@ -159,6 +160,15 @@ impl<'a> From<Arc<str>> for Str<'a> {
     }
 }
 
+impl<'a> From<Cow<'a, str>> for Str<'a> {
```

Comment:
this change is also a separate logical change, especially since it's a different crate.